src/server/helper.ts

```diff
@@ -116,6 +116,29 @@ class Helper {
       return '';
     return '\n' + '='.repeat(20) + ' Browser output: ' + '='.repeat(20) + '\n' + logs.join('\n');
   }
+
+  static parseRemoteAddress(value?: string) {
```

Comment:
This method doesn't work with [::1]:80. I think parsing ipv6 addresses is somewhat complicated. I'd do `new URL('http://' + address)`. Note that if the port is empty is actually 80.

Reply:
WK returns addresses like `::1.8910` which explode explode with `new URL('http://' + address)`

Reply:
oh no :(. ok if its not a valid ip then i guess its fine